src/app/src/components/Navbar.jsx

```diff
@@ -84,7 +84,7 @@ export default function Navbar() {
                     </Link>
                 </div>
                 <div id="google_translate_element" />
-                <NavbarLoginButtonGroup />
+                <Route component={NavbarLoginButtonGroup} />
```

Comment:
Confirming that we are using <Route> here so that we get access to the history object in the component lifecycle methods.

Reply:
Yeah, that's right. I also tried out using the `withRouter` decorator but I needed `history.push` to be available in the `mapStateToDispatch` function, which didn't seem to happen with `withRouter`.